### PR TITLE
Add a topic about BZFlag

### PR DESCRIPTION
**Pull request by TheGiraffe3**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created either the topic I am modifying/adding or the collection entry I am modifying/adding.

### Which change are you proposing?

  - [x] Curating a new topic or collection

---

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> BZFlag is a popular game from the 90s.

### CODE REVIEW

File: topics/bzflag/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+display_name: BZFlag
+logo: bzflag.png
+short_description: 3D multiplayer tank game created in 1992.
+topic: bzflag
+url: https://bzflag.org
+wikipedia_url: https://en.wikipedia.org/wiki/BZFlag
+---
+Everyone knows that having a tank that can _just_ fly is pretty boring. What if we told you tanks could be equipped with super powers? You'd think we're crazy, right? We were so busy thinking on how we could, that we didn't stop to think whether or not we should. Super powers for all your tanks!
+
+BZFlag is a 3D multiplayer tank game first made in 1992. It runs on all three major platforms and was chosen for the Google Summer of Code three times, in 2007, 2008, and 2009.
```
Comment (kenyonj) [inline topics/bzflag/index.md:11]: While I think this is fun, I think we should avoid editorializing here.
```suggestion
BZFlag is a 3D multiplayer tank game first made in 1992. It runs on all three major platforms and was chosen for the Google Summer of Code three times, in 2007, 2008, and 2009.
```

**Review by kenyonj — CHANGES_REQUESTED** [review]

File: topics/bzflag/index.md (hunk 1 above)
Comment (TheGiraffe3) [reply topics/bzflag/index.md:11]: I was trying to copy the website's "description", but it makes sense not to.

**Review by DemonBarber1020 — COMMENTED** [review]
download
My logo of an amazing and potentially a great unique idea... as for myself being a Barber and hairstylist I came up with this new exclusive clothing brand only made for the barbers and hairstylists also cosmetologist etc

**Review by jmeridth — APPROVED** [review]

**Comment by jmeridth** [comment]
@kenyonj Looks like the changes were applied.  Can you please re-review.  Thank you.

**Review by kenyonj — APPROVED** [review]

**Pull request merged**